packages/apollo-language-server/src/project/client.ts

```diff
@@ -271,15 +272,19 @@ export class GraphQLClientProject extends GraphQLProject {
     await this.loadingHandler.handle(
       `Loading Engine data for ${this.displayName}`,
       (async () => {
-        const {
-          schemaTags,
-          fieldStats
-        } = await engineClient.loadSchemaTagsAndFieldStats(serviceID);
-        this._onSchemaTags && this._onSchemaTags([serviceID, schemaTags]);
-        this.fieldStats = fieldStats;
-        this.lastLoadDate = +new Date();
-
-        this.generateDecorations();
+        try {
+          const {
+            schemaTags,
+            fieldStats
+          } = await engineClient.loadSchemaTagsAndFieldStats(serviceID);
+          this._onSchemaTags && this._onSchemaTags([serviceID, schemaTags]);
+          this.fieldStats = fieldStats;
+          this.lastLoadDate = +new Date();
+
+          this.generateDecorations();
+        } catch (e) {
+          console.error(e);
```

Comment:
Just wrapped this in a try/catch so we're not letting any unhandled rejections through

Reply:
Just making a note that I believe that errors here are meant to bubble up to the LoadingHandler. They're try/catch in the handler so it can communicate them to the extension. This may be a better UX or we may need to improve the messages we send across the connection.

Reply:
This has to be here or else I get an unhandled rejection error 😬 I'd love to know how to properly fix this though